src/pk-transaction.c

```
@@ -4274,6 +4290,80 @@ pk_transaction_remove_packages (PkTransaction *transaction,
 	pk_transaction_dbus_return (context, error);
 }
 
+static void
+pk_transaction_purge_packages (PkTransaction *transaction,
```

Clang (14.0.6)

[warning] 3 adjacent parameters of 'pk_transaction_purge_packages' of similar type ('int *') are easily swapped by mistake

(bugprone-easily-swappable-parameters)